hmailserver/source/Server/hMailServer.Minidump/hMailServer.Minidump.vcxproj

```diff
@@ -51,7 +51,7 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <ClCompile>
       <Optimization>Disabled</Optimization>
-      <AdditionalIncludeDirectories>$(hMailServerLibs)\boost_1_72_0;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(hMailServerLibs)\boost_1_73_0;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
This change has no relation to the "Installer bug fixes" right?

Reply:
No! Sorry i was using a newer Boost version for my own test builds ;d